.appveyor.yml

```diff
@@ -38,6 +46,9 @@ artifacts:
   - path: qgroundcontrol-installer.exe
     name: qgcinstaller
 
+  - path: symbols\**\*.*_
```

Comment:
@NaterGator why is the pdb file called `qgroundcontrol.pd_` and the exe `qgroundcontrol.ex_`?

Reply:
Because they are compressed for use with the Pepto symbol server (which I'm not really sure how to set up).

Reply:
Seems like it might be easier to store the artifacts uncompressed and then we could just pull individual versions from appveyor as needed?